script.js

```diff
@@ -0,0 +1,6 @@
+const menuBtn = document.querySelector('.hamburger');
+const menuNav = document.querySelector('.mobile-nav');
+menuBtn.addEventListener('click', () => {
+  menuBtn.classList.toggle('is-active');
+  menuNav.classList.toggle('is-active');
+});
```

Comment:
- [ ] Please add a CSS rule that closes horizontal scrolling when the mobile menu is on.